kuegi_bot/bots/trading_bot.py

```diff
@@ -26,20 +26,20 @@ def __init__(self, logger, directionFilter: int = 0):
         self.myId = "GenericBot"
         self.logger = logger
         self.directionFilter = directionFilter
-        self.order_interface: OrderInterface = None
+        self.order_interface: ExchangeInterface = None
         self.symbol: Symbol = None
         self.unique_id: str = ""
         self.last_time = 0
-        self.last_tick_time : datetime = None
+        self.last_tick_time: datetime = None
         self.is_new_bar = True
-        self.open_positions = {}
+        self.open_positions : Dict[str,Position] = {}
```

Comment:
minor code-styling: to be consistent with other properties in here. no leading whitespace before :